Makefile

```diff
@@ -13,7 +13,7 @@
 # limitations under the License.
 
 # ENVTEST_K8S_VERSION refers to the version of kubebuilder assets to be downloaded by envtest binary.
-ENVTEST_K8S_VERSION = 1.29.1
+ENVTEST_K8S_VERSION = 1.31.1
```

Comment:
Latest available is 1.31.0, and please drop the custom bucket from the envtest invocation

Reply:
Ahh, that'll be it 🤦